packages/driver/cypress/integration/e2e/multi-domain/commands/multi_domain_screenshot.spec.ts

```diff
@@ -0,0 +1,183 @@
+// @ts-ignore / session support is needed for visiting about:blank between tests
+context('screenshot specs', { experimentalSessionSupport: true, experimentalMultiDomain: true }, () => {
+  beforeEach(() => {
+    cy.visit('/fixtures/multi-domain.html')
+    cy.get('a[data-cy="screenshots-link"]').click()
+
+    this.serverResult = {
+      path: '/path/to/screenshot',
+      size: 12,
+      dimensions: { width: 20, height: 20 },
+      multipart: false,
+      pixelRatio: 1,
+      takenAt: new Date().toISOString(),
+      name: 'name',
+      blackout: [],
+      testAttemptIndex: 0,
+      duration: 100,
+    }
```

Comment:
```suggestion
    this.serverResult = {
      path: '/path/to/screenshot',
      size: 12,
      dimensions: { width: 20, height: 20 },
      multipart: false,
      pixelRatio: 1,
      takenAt: new Date().toISOString(),
      name: 'name',
      blackout: [],
      testAttemptIndex: 0,
      duration: 100,
    }

    cy.visit('/fixtures/multi-domain.html')
    cy.get('a[data-cy="screenshots-link"]').click()
```

Just a preference, since this is the order in which the code executes.

Reply:
Updated [78540cc](https://github.com/cypress-io/cypress/pull/20119/commits/78540ccb3b2eea5cd70961ab50d20f7367614da7)